src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -712,6 +712,9 @@ export default class ConvasController {
         if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
             return;
         }
+        if (GameController.game.option.get('drawIlsDistanceSeparator') === 'no') {
```

Comment:
This line should use the `GAME_OPTION_NAMES` constant here instead of it's literal string value.